packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -762,11 +762,20 @@ class FlutterPlugin implements Plugin<Project> {
     }
 
     /**
-     * Returns `true` if the given path contains an `android/build.gradle` file.
+     * Returns `true` if the given path contains an `android` directory
+     * containing a `build.gradle` or `build.gradle.kts` file.
      */
     private Boolean doesSupportAndroidPlatform(String path) {
-        File editableAndroidProject = new File(path, 'android' + File.separator + 'build.gradle')
-        return editableAndroidProject.exists()
+        File buildGradle = new File(path, 'android' + File.separator + 'build.gradle')
+        File buildGradleKts = new File(path, 'android' + File.separator + 'build.gradle.kts')
+        if (buildGradle.exists() && buildGradleKts.exists()) {
+            logger.error(
```

Comment:
I think the `project` from 370f40e6dfbf742d7b980dc7cc38152975ed8d84 is missing again. I readded it in #142498

Reply:
ugh, that's right! thank you

wow I wish so much for Groovy's removal in our codebase